bitrise.yml

```diff
@@ -20,9 +20,9 @@ workflows:
     - golint:
     - errcheck:
     - go-test:
-    - certificate-and-profile-installer:
```

Comment:
Why comment this out?

Reply:
weird codesigning issues with downloadsing .p12.

i will push another version without comment